### ISSUE TITLE
suggestion: custom branding icon patch

### ISSUE THREAD

**Issue by andylee516** [anddea/revanced-patches#747]
### Application

YouTube

### Suggestion description

[patch_log.txt](https://github.com/user-attachments/files/16348616/patch_log.txt)
This is patchlog from 2.227.0 dev24. However, before you release stable 2.227.0, there is something you should know. I had patched same patchoption(including same custom header zip file) in inotia rvx and your rvx. In inotia, it had no problem patching but your rvx caused error at last despite clearing patch option.
Related to header zip file, i had duplicated files and named premium as you told me at last time.
I will leave my header zip file i had used, so can you test that using that zip file works successfully?
[extended.zip](https://github.com/user-attachments/files/16348694/extended.zip)

### Acknowledgements

- [X] This issue is not a duplicate of an existing suggestion.
- [X] This is not a feature request.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by AbakNacchan** [comment]
Custom Branding Icon resources from Anddea is different from inotia00.

**Comment by anddea** [comment]
What custom icon did you use? The issue might be in the icon.

**Comment by andylee516** [comment]
> What custom icon did you use? The issue might be in the icon.

youtube original icon

**Comment by anddea** [comment]
> youtube original icon

Yes, it has issues, if you want original icon, you can disable this patch and it will use the same original. I'll fix it later.

**Comment by andylee516** [comment]
> > youtube original icon
> 
> Yes, it has issues, if you want original icon, you can disable this patch and it will use the same original. I'll fix it later.

if disabling this patch, then we can't use custom header, am i right?

**Comment by anddea** [comment]
> if disabling this patch, then we can't use custom header, am i right?

Yes

**Comment by andylee516** [comment]
> > if disabling this patch, then we can't use custom header, am i right?
> 
> Yes

thank you for fixing bug
